Cython/Includes/cpython/datetime.pxd

```diff
@@ -141,6 +149,9 @@ cdef extern from "datetime.h":
 cdef inline void import_datetime():
     PyDateTime_IMPORT
 
+# Time zone singleton for UTC.
+cdef object utc_tz = PyDateTimeAPI.TimeZone_UTC
```

Comment:
This can't work because we haven't imported the C-API yet at this point, i.e. the user did not call `import_datetime()` yet.

Also, it requires a cast to `<object>`.

Reply:
Ah! That's what I was missing, so make that a function like `get_utc()` and then do that cast. Gotcha